test/test-supervisor.js

```diff
@@ -19,20 +22,20 @@ function once(fn) {
   };
 }
 
-describe('supervisor', function(done) {
+tap.test('supervisor', function(t) {
   var child;
 
-  this.timeout(10000); // CI machines are slow for process creation
+  // FIXME this.timeout(10000); // CI machines are slow for process creation
```

Comment:
This probably isn't even needed with tap, since these tests will now honour the timeout given via CLI.